vj4/ui/templates/domain_manage_user.html

```diff
@@ -80,6 +81,9 @@ <h1 class="section__title">{{ _('{0}: Users').format(handler.domain['name']) }}<
           <td class="col--user">
             {{ user.render_inline(udict[rudoc['uid']], badge=false) }}
           </td>
+          <td class="col--name-alias">
+            {{ rudoc['name_alias'] if 'name_alias' in rudoc else '' }}
```

Comment:
`rudoc['name_alias']|default('')`?

Reply:
Please note the expression between `{{` and `}}` is not Python :)